README.md

```diff
@@ -2666,6 +2666,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Vaishnavi Rastogi](https://github.com/Navi68?)
   - [Vallabh Tiwari](https://github.com/vallabhtiwari)
   - [Vamsi Ravi](https://github.com/vamsi963601)
+  - [Varun Tanjore](https://github.com/varun-TR)
```

Comment:
Please keep this document in the correct alphabetical order. Your name should actually be in line 2677.